server/service/src/purchase_order_line/insert/mod.rs

```diff
@@ -28,15 +38,22 @@ pub struct InsertPurchaseOrderLineInput {
 
 pub fn insert_purchase_order_line(
     ctx: &ServiceContext,
-    store_id: &str,
     input: InsertPurchaseOrderLineInput,
 ) -> Result<PurchaseOrderLineRow, InsertPurchaseOrderLineError> {
     let purchase_order_line = ctx
         .connection
         .transaction_sync(|connection| {
-            validate(store_id, &input, connection)?;
+            let validate_input = validate::ValidateInput {
+                id: input.id.clone(),
+                purchase_order_id: input.purchase_order_id.clone(),
+                item_id: input.item_id.clone(),
+                // TODO amend default value if we extend standard insert line input
+                requested_pack_size: 0.0, // Default value
```

Comment:
Later we might want to allow more values inputed in insert lines - so shouldn't default to 0.0.

As per above - users can't add duplicate item codes with the same pack size - so they would need to edit their first code to change the pack size before adding another of the same item code.